packages/dart/sshnoports/bin/demo_socket_client.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:io';
+
+/// - simple client which talks to a demo_socket_server
+/// - sends a request for N kBytes of data to be sent to us
+/// - listens for the data and writes a message at the end
+/// saying how much it received vs expected
```

Comment:
added this and demo_socket_server.dart to facilitate highly concurrent load testing of npt